### PR TITLE
Add order model

### PR DESCRIPTION
**Pull request by denysokhrymovych**

### CODE REVIEW

File: src/main/java/com/example/bookshop/model/Order.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    @JoinColumn(nullable = false, name = "user_id")
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    private User user;
+    @JdbcTypeCode(SqlTypes.VARCHAR)
+    private Status status;
+    @Column(nullable = false)
+    private BigDecimal total;
+    @CreationTimestamp
+    private LocalDateTime orderDate;
```
Comment (d1sam) [inline src/main/java/com/example/bookshop/model/Order.java:46]:
```suggestion
    @Column(nullable = false)
    private LocalDateTime orderDate;
```

File: src/main/java/com/example/bookshop/model/Order.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(nullable = false, name = "user_id")
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    private User user;
+    @JdbcTypeCode(SqlTypes.VARCHAR)
+    private Status status;
```
Comment (d1sam) [inline src/main/java/com/example/bookshop/model/Order.java:41]:
```suggestion
   @Column(nullable = false)
    private Status status;
```

File: src/main/java/com/example/bookshop/model/OrderItem.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    private Order order;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "book_id")
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    private Book book;
+    @Column(nullable = false)
+    private int quantity;
```
Comment (d1sam) [inline src/main/java/com/example/bookshop/model/OrderItem.java]:
```suggestion
    private Integer quantity;
```
it's better to use wrappers as fields inside Entity

**Review by d1sam — CHANGES_REQUESTED** [review]
Good job, but there are some changes that should be done!

**Review by vovasalyha — APPROVED** [review]

**Pull request merged**